doc/releases.md

```diff
@@ -0,0 +1,200 @@
+
+The GPG keys should be fetchable from a known third-party keyserver, currently the SKS Keyservers at <https://sks-keyservers.net> are recommended. Use the [submission](https://sks-keyservers.net/i/#submit) form to submit a new GPG key. Keys should be fetchable via:
+
+```
+gpg --keyserver pool.sks-keyservers.net --recv-keys <FINGERPRINT>
+```
+
+Additionally, full GPG key fingerprints for individuals authorized to release should be listed on the io.js GitHub README.md file.
+
+## How to create a release
+
+Notes:
+
+ - Dates listed below as _"YYYY-MM-DD"_ should be the date of the release **as UTC**. Use `date -u +'%Y-%m-%d'` to find out what this is.
+ - Version strings are listed below as _"vx.y.z"_, substitute for the release version.
+
+### 1. Ensure that HEAD Is Stable
+
+Run a **[iojs+any-pr+multi](https://jenkins-iojs.nodesource.com/job/iojs+any-pr+multi/)** test run to ensure that the build is stable and the HEAD commit is ready for release.
+
+### 2. Produce a Nightly Build _(optional)_
+
+If there is reason to produce a test release for the purpose of having others try out installers or specifics of builds, produce a nightly build using **[iojs+release+nightly](https://jenkins-iojs.nodesource.com/job/iojs+release+nightly/)** and wait for it to drop in <http://iojs.org/download/nightly/>.
+
+This is particularly recommended if there has been recent work relating to the OS X or Windows installers as they are not tested in any way by CI.
+
+### 4. Update the _CHANGELOG.md_
+
+Use the following git command to produce a list of commits since the last release:
+
+```
+git log --pretty=format:"* [%h] - %s (%aN)" \
+  --since="$(git show -s --format=%ad `git rev-list --tags --max-count=1`)"
+```
+
+_(You will probably need to omit the first two commits as they relate to the last release.)_
+
+The _CHANGELOG.md_ entry should take the following form:
+
+```
+## YYYY-MM-DD, Version x.y.z, @user
+
+### Notable changes
+
+* List interesting changes here
+* Particularly changes that are responsible for minor or major version bumps
+* Also be sure to look at any changes introduced by dependencies such as npm
+* ... and include any notable items from there
+
+### Known issues
+
+* Include this section if there are any known problems with this release
+* Scan GitHub for unresolved problems that users may need to be aware of
+
+### Commits
+
+* Include the full list of commits since the last release here
+```
+
+### 5. Update _src/node_version.h_
+
+The following macros should already be set for the release since they will have been updated directly following the last release. They shouldn't require changing:
+
+```
+#define NODE_MAJOR_VERSION x
+#define NODE_MINOR_VERSION y
+#define NODE_PATCH_VERSION z
+```
+
+However, the `NODE_VERSION_IS_RELEASE` macro needs to be set to `1` for the build to be produced with a version string that does not have a trailing pre-release tag:
+
+```
+#define NODE_VERSION_IS_RELEASE 1
+```
```

Comment:
This section should explain when to bump NODE_MODULE_VERSION and what it means for the version number (at least a minor bump; I don't think we've decided on a major bump policy yet.)